### DISCUSSION TITLE
Question:  DatePicker Styling

### DISCUSSION THREAD

**Discussion by hooligan495** [microsoft/microsoft-ui-xaml#7507] [Q&A]
Hi, 
I am trying to change the styling of the DatePicker's FlyoutPresenter.. specifically I'd like to change the hover and pressed states when you move your mouse pointer over a Month or Day or Year.. Currently in the "dark" theme this hover effect is a gray background.  I'd like to change it to a colored back ground (My UX team is needy) and the Pressed state seems to be a lighter gray.  And I also need to make this a color.  However it going through the styles in the Visual Tree in VS I'm seeing things like LoopingSelector and LoopingSelectorItem.  Is tehre a good way to figure out what style corresponds to what I want to change? or Am I always left with hunting and pecking (and don't take this last statement as a negative... I don't mind the hunting.. it's just having a decoder ring to go with it would be cool.. I can't seem to find any source code that might help me).  

I am using Windows.Xaml.UI 2.2 if that helps (we hope to be upgrading to newer versions soon but it's not in the cards yet).

Thanks in advance.  And thanks for everyone's work on this project!

**Comment by YuliKl** [comment]
@hooligan495 I would begin the hunt in DatePicker's themeresources files: https://github.com/microsoft/microsoft-ui-xaml/tree/master/dev/DatePicker
